src/libraries/System.Net.Quic/src/System/Net/Quic/Interop/QUIC_SETTINGS.IEquattable.cs

```diff
@@ -0,0 +1,95 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Quic
+{
+    internal partial struct QUIC_SETTINGS : System.IEquatable<QUIC_SETTINGS>
```

Comment:
Theoretically, you could move this into System.Net.Quic/src/System/Net/Quic/Internal/MsQuicExtensions.cs where we already have something similar for `ToString`. Depends, whether you expect the MsQuic generator ask to get closed soonish or rather not...

Reply:
it does not seem like they will implement it soon, so I will move this